sccm-ps/ConfigurationManager/New-CMTSRule.md

```diff
@@ -0,0 +1,250 @@
+```
+
+## Description
+
+## EXAMPLES
+
+### Example 1: Sets a task sequence variable to a value
+```
+PS C:\> New-CMTSRule -Variable @{"MyTSVar" = 12345678}
+```
+
+## PARAMETERS
+
+### -AssetTag
+
+
+```yaml
+Type: String
+Parameter Sets: ComputerCondition
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultGateway
+
+
+```yaml
+Type: String
+Parameter Sets: LocationCondition
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -MacAddress
+
+
+```yaml
+Type: String
+Parameter Sets: ComputerCondition
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Make
+
+
+```yaml
+Type: String
+Parameter Sets: MakeModelCondition
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Model
+
+
+```yaml
+Type: String
+Parameter Sets: MakeModelCondition
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ReferencedVariableName
+
+
+```yaml
+Type: String
+Parameter Sets: VariableCondition
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ReferencedVariableOperator
+
+
+```yaml
+Type: String
+Parameter Sets: VariableCondition
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ReferencedVariableValue
+
+
+```yaml
+Type: String
+Parameter Sets: VariableCondition
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -SerialNumber
+
+
+```yaml
+Type: String
+Parameter Sets: ComputerCondition
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Uuid
+
+
+```yaml
+Type: String
+Parameter Sets: ComputerCondition
+Aliases: 
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Variable
+
+
+```yaml
+Type: Hashtable
+Parameter Sets: (All)
+Aliases: Variables
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
```

Comment:
If these remaining sections are blank, let's just remove them. Or can comment them out: `<!-- stuff in here -->`